moped-editor/src/views/projects/NewProjectView.js

```diff
@@ -207,6 +207,8 @@ const NewProjectView = () => {
   const [success, setSuccess] = React.useState(false);
   const timer = React.useRef();
 
+  console.log(success);
+
```

Comment:
...Rid?

Reply:
yeah good call. the `success` var wasn't being used so I just added a console.log as a temp fix to fix the breaking build. But I think the better solution is just to remove it entirely.